@xen-orchestra/lite/src/stories/web-core/card-numbers.story.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties }"
+    :params="[
+      prop('label').type('string').preset('Label').widget(),
+      prop('value').type('number').preset(0).widget(),
+      prop('unit').type('string').preset('GB').widget(),
+      prop('size').type('string').enum('small', 'medium').preset('small').widget(),
+      prop('max').type('number').widget(),
+    ]"
+    :presets="presets"
```

Comment:
- `presets` is undefined
- missing `required()` on required props

```suggestion
    :params="[
      prop('label').str().required().preset('Label').widget(),
      prop('value').num().required().preset(0).widget(),
      prop('size').str().required().enum('small', 'medium').preset('small').widget(),
      prop('unit').str().preset('GB').widget(),
      prop('max').num().widget(),
    ]"
```